product_code_builder/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+#    @author Abdessamad HILALI <abdessamad.hilali@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Product Code Builder",
+    'author': "Akretion, Odoo Community Association (OCA)",
+    'summary': "Create product references according to attributes",
+    'website': "https://www.akretion.com",
+    'license': 'AGPL-3',
+    'category': 'Product',
+    'version': '9.0.1.0.0',
+    'depends': [
+        'product',
```

Comment:
@bealdav Sorry for the late reply... Both are required since I extends the views defined in product and therefore I have a direct dependency on product. In case of direct dependency it's always better to avoid to rely on the fact that this dependency is transitively acquired by an other one.